taskcluster/ci/toolchain/qt.yml

```diff
@@ -5,7 +5,7 @@
 task-defaults:
     worker:
         env:
-            QT_VERSION: "6.2.4"
+            QT_VERSION: "6.3.2"
             QT_MAJOR: "6.2"
```

Comment:
QT_MAJOS is 6.3 then?

Reply:
Ok so, I added the QT vesions to the scripts themselves. The decision taks fails if I leave this environment empty though... Do you know why that is? The error is in the taskcluster code itself, but as far as I can tell, the env is not required in the worker section. Let me know.